app/editor/src/features/admin/evening-overview/interfaces/IEveningOverviewItemType.ts

```diff
@@ -0,0 +1,12 @@
+import { OptionItem } from 'tno-core';
+
+export enum EveningOverviewItemType {
```

Comment:
The practice we have is to place enums in the constants folder.  We also include both enum structures to handle any possible situation.

`EveningOverviewItemType` would be an enum with integer values.
`EveningOverviewItemTypeName` would be an enum with string values.

Refer to the existing implementation to keep things consistent.